css/css.md

```diff
@@ -37,6 +40,7 @@ About 4.5 hours
 - Effectively use the `display` property
 
 ### Specific Things To Teach
+
```

Comment:
Thanks for double-checking, the sections do look good.  Would you please move supplemental resources to the end of the outline, after check for understanding?

Reply:
Also change the word "materials" to "resources".

Reply:
Should I also move the supplemental resources part after check for understanding in the other files?

Reply:
Yes, thank you!